addons/account/models/account_move_line.py

```diff
@@ -2973,3 +2973,22 @@ def _get_downpayment_lines(self):
         This method is overridden in the sale order module.
         '''
         return self.env['account.move.line']
+
+    # -------------------------------------------------------------------------
+    # PLACEHOLDERS
+    # -------------------------------------------------------------------------
+
+    def _predict_product(self):
```

Comment:
Since everything regarding the prediction is in `account_accountant` now, wouldn't it be easier to just override `_predict_taxes` in `l10it_reports` instead of these ugly placeholders?
Would be **a lot** more explicit about what this commit is doing too...

Reply:
We cannot because of the order of things: the prediction is done on the on_change of the name and this comes when we create and post the move, AFTER we got the information from the XML, and the prediction actively overwrites the tax information we imported correctly.

The prediction is actually a key feature for **product import**. No one is ever gonna bill you with the exact description you have in Odoo. 

The **taxes** instead MUST be the ones in the XML. We have all information about that and they cannot really differ between the XML and the bill - otherwise you'll pay a wrong amount, and mess up with the Tax Agency.

That's why we disable the "normal prediction" and do it our own way, and we need context key and placeholders. I don't know how disturbing can these placeholders or an explicit `if hasattr` can be... if we want the feature, something must go there.

Also, I would add something in the chatter if we predicted something, and absolutely not be silent about that: "We've seen the product description is "Whiteboard pen red" in the XML and we matched against the "Whiteboard pen - color red" product in Odoo" - this would probably save time to people who always order the same thing and don't want to update partner product descriptions.

@chklop @jco-odoo @sold-odoo